description/launch/simulate_robot.launch

```diff
@@ -0,0 +1,12 @@
+<launch>
+    
+    <arg name="use_gui" default="true" />
+    
+    <!-- Run Gazebo -->
+    <include file="$(find gazebo_ros)/launch/empty_world.launch" >
+        <arg name="gui" value="$(arg use_gui)" />
+    </include>
```

Comment:
the contents of `iarrc_worlds.launch` should be here (I should be simulate the robot with the world I want in this file, currently this file only lets me run empty.world)

Reply:
Both iarrc_worlds and simulate_robot.launch call spawn_robot.launch. Should I delete one or the other?